scripts/tokenBridgeDeployment.ts

```diff
@@ -136,8 +132,7 @@ export const deployErc20l2 = async (rs: RuntimeState, deployer: Signer) => {
     : await deployBehindProxy(deployer, L1WethGateway__fac, proxyAdmin.address)
   rs.l2.wethGateway = wethGateway.address
 
-  const weth = wethAddress
-  rs.l2.weth = weth
+  const weth = rs.l2.weth
```

Comment:
```suggestion
  const weth = rs.l2.weth
		? AeWETH__fac.attach(rs.l2.weth).connect(deployer)
		: await deployBehindProxy(deployer, AeWETH__fac, proxyAdmin.address)
	  rs.l2.weth = weth.address
```
(Adding potential missing code)
Should this try to deploy if the contract doesn't exist?